eng/WpfArcadeSdk/tools/CodeGen/AvTrace/GenTraceSources.targets

```diff
@@ -0,0 +1,26 @@
+<Project>
+  <PropertyGroup>
+    <WindowsBaseAvTraceMessageFile>$(WpfSourceDir)\WindowsBase\AvTraceMessages.json</WindowsBaseAvTraceMessageFile>
```

Comment:
Could this be an opt-in? For e.g., if a project sets `$(AvTraceMessagesFile)`, then this target picks up the value and uses it? 

Alternatively, could this fit into `GenTraceSources.props` using a pattern somewhat similar to what is in use within `ShippingProjects.props`, for e.g., like this: 

```XML
<AvTraceEnabledProjects>
  WindowsBase;
  PresentationCore;
  PresentationFramework
  </AvTraceEnabledProjects>
```

In GenTraceSources.targets, 




```XML
<AvTraceMessageFile Condition="$(AvTraceEnabledProjects.Contains('$(NormalizedMSBuildProjectName)'))>$(MSBuildProjectDirectory)AVTraceMessages.json</AvTraceMessageFile>
```

Reply:
Yeah i had tried that initially (or some variation of it), there is such a headache with ordering of targets that i decided to have the projects manually include the targets that are needed

Reply:
Yeah i had tried that initially (or some variation of it), there is such a headache with ordering of targets that i decided to have the projects manually include the targets that are needed

Reply:
but maybe i can revisit it now that i have everything working and find a way to make it work

Reply:
i'm going to leave as is, i think it's cleaner that these targets aren't magically imported. couple that with the dependency on the ordering of including the Sdk.targets from Microsoft.NET.Sdk, this will easier to maintain